src/BloomExe/MiscUI/ProblemReporterDialog.cs

```diff
@@ -470,10 +469,34 @@ private string GetFullDescriptionContents(bool appendLog)
 			bldr.AppendLine("=Problem Description=");
 			bldr.AppendLine(_description.Text);
 			bldr.AppendLine();
+			GetAdditionalEnvironmentInfo(bldr);
 			GetStandardErrorReportingProperties(bldr, appendLog);
 			return bldr.ToString();
 		}
 
+		private void GetAdditionalEnvironmentInfo(StringBuilder bldr)
+		{
+			if (Book == null)
+				return;
```

Comment:
The main problem is that introducing knowledge of WorkspaceView to ProblemReporterDialog might produce a circular reference. WorkspaceView at least has the Collection Name string.
I also spent way too much time trying to make tests that didn't require UI... unfruitful.